src/main.js

```diff
@@ -376,6 +376,32 @@ ${note}
 ` : ''}`).join('')}
 `
 
+const exportToBibTex = ({ annotations, metadata }) => {
+    // Escape all Tex characters that BibTex requires
+    const esc = cont => !cont ? '' : Array.from(cont).map(c =>
+        c === '#' ? '\\#'
+        : c === '$' ? '\\$'
+        : c === '%' ? '\\%'
+        : c === '&' ? '\\&'
+        : c === '\\' ? '\\textbackslash{}'
+        : c === '^' ? '\\textasciicircum{}'
+        : c === '_' ? '\\_'
+        : c === '{' ? '\\{'
+        : c === '}' ? '\\}'
+        : c === '~' ? '\\textasciitilde{}'
+        : c).join('')
+
+    // Math functions needed to avoid Tex problems
+
+    return annotations.map(({ text, note }, i) =>
+`@book{ref${Math.round(Math.random() * 10000)}:${i + 1},
+    author = {${esc(metadata.creator) || 'unknown'}},
+    publisher = {${esc(metadata.publisher) || 'unknown'}},
+    year = {${metadata.pubdate ? metadata.pubdate.slice(0, 4) : 'unknown'}},
+    note = {${_('Quote: ') + esc(text) + (note ? _(" Note: ") + esc(note) : '')}}
+},
+`).join('')
```

Comment:
Maybe put the comma and new line in `join` so there won't be a trailing comma at the end? Although I'm not familiar with BibTeX so I don't know if that would be good.

Reply:
BibTeX can ignore this, and probably will.